tests/UKHO.ERPFacade.API.FunctionalTests/FunctionalTests/BulkPriceUpdateScenario.cs

```diff
@@ -55,5 +55,17 @@ public async Task WhenValidEventReceivedWithInvalidToken_ThenBPUpdateReturns403U
 
         }
 
+        [Category("DevEnvFT")]
+        [Test, Order(1)]
+        [TestCase("PC1_Payload.JSON", TestName = "WhenValidEventReceivedWithValidToken_ThenBPUpdateReturn200OkResponse")]
```

Comment:
Can be renamed as WhenValidBulkPriceEventReceivedWithValidToken_ThenBPUpdateReturns200OkResponse